### DISCUSSION TITLE
Can't send verification

### DISCUSSION THREAD

**Discussion by XxdonlukexX** [community/community#86616] [GitHub Education]
### Hi everyone,

I got rejected for github student dev pack a few days ago. Now I tried to send a better picture on the same website again, but after I upload the picture and clikc on "Process my application" it just turns to "Processing Information" without ever changing. The loading bar doesn't even show up. The first time it worked well but now I got these problems. Does anyone know, how to fix that?

**Comment by dariohsb** [comment]
Clear your cookies/cache and disable adblock if you have one, after that, try again.

**Reply by XxdonlukexX** [reply]
Thanks for the fast response,
I already did that but it did not work. It still keeps "Processing"...
Edit:
And I don't get, why it worked like 3-4 days ago but now it doesn't. I used the same browser, addons etc.
